src/config.js

```diff
@@ -62,6 +62,18 @@ const listeDomain = () => {
   }
 };
 
+const ndlaFrontendDomain = () => {
+  console.log('env,', process.env.NDLA_ENVIRONMENT);
+  switch (process.env.NDLA_ENVIRONMENT) {
+    case 'local':
+      return 'http://localhost:30017';
+    case 'prod':
+      return 'https://ndla.no';
+    default:
+      return `https://${process.env.NDLA_ENVIRONMENT}.ndla.no`;
+  }
+};
```

Comment:
Vil dette virke å bruke direkte i h5pPlugin i article-converter? @gunnarvelle 
prøvde å gjøre det raskt men på default får jeg at det blir `undefined.ndla.no`

Reply:
nvm, så jeg har brukt feil variabelnavn

Reply:
Denne blir bare _undefined_. Prøver å bruke den i `h5pPlugin.js` linje 22, men får _undefined_.ndla.no

Reply:
Har du satt NDLA_ENVIRONMENT? Sett den til test lokalt så ser du om det fungerer.

Reply:
@gunnarvelle det feiler når jeg setter den til test (i ndla-frontend: `throw new Error(errors.unsupportedEnvironment())`)

ser at der ser ting litt annerledes ut og har 
`    case 'dev':
      return 'https://test.ndla.no';`
      og
`const ndlaEnvironment = getEnvironmentVariabel('NDLA_ENVIRONMENT', 'dev');`
      
Koden her i pr har jeg kopiert fra editorial som ser sånn her ut som koden jeg har kommentert. 

Skjønner ikke helt hva som er forskjell, og hva jeg må gjøre for å få det til å funke. Kanskje legge til sjekk for dev slik som i ndla-frontend? Eller er det en annen løsning for å få det til å virke med ndla-frontend kjørende med `yarn start-with-local-graphql-and-article-converter`.

Reply:
I ndla-frontend defaulter den til dev dersom parameteren ikkje er satt, og dersom du har dev så brukes test. samme resultatet skulle du fått med NDLA_ENVIRONMENT satt til test. Kvar kjem error fra? Kan ikkje finne unsupportedEnvironment i koden.

Reply:
er noe sass opplegg? 
`..../node_modules/node-sass/lib/binding.js`

`Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (83)`
så da er kanskje bare spesifikt for mitt OS da.. 👩🏻‍💻🙃 Det er kanskje på tide å oppdatere os'et uansett.